src/bin-driver.rs

```diff
@@ -0,0 +1,15 @@
+use anyhow::{anyhow, Result};
+use cargo_gccrs::{Error, Gccrs};
+
+fn main() -> Result<()> {
+    Gccrs::maybe_install()?;
+
+    let args: Vec<String> = std::env::args().collect();
+
+    let res = match args.get(1).map(String::as_str) {
+        Some("rustc") => Gccrs::handle_rust_args(&args),
```

Comment:
A function call named `handle_rust_args` that also compiles things is a bit confusing IMO.

Reply:
That's fair. Do you have any idea as to what to call the function? Since it doesn't only compile, and also has the task of translating the Rust arguments

Reply:
I would either split it up and first handle the args and then compile or just call it ``compile`` if you want to keep only one public function to interface with the lib part.